.github/ISSUE_TEMPLATE/enhancement_proposal.md

```diff
@@ -16,15 +16,12 @@ When would you use this?
 **Message from the maintainers**:
 
 Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.
-<!-- 
 
+<!-- 
```

Comment:
Can you also change all "prioritise" in these templates to "prioritize" since you are using the second one?

Reply:
consistency is good - will do